apps/cli/src/commands/send.ts

```diff
@@ -81,7 +81,6 @@ export default {
                           deviceId: opts.device || "",
                         })
                         .pipe(
-                          map(toSignOperationEventRaw),
```

Comment:
would it be ok to send a separate PR for this change? I'm not sure to understand it myself and would be good to have a dedicated PR of this "fix" (if it's a bug, please explain as it went under our radar 🙏 ) thanks

Reply:
Hi gre, long time no interaction! It is good to work again on a PR haha.

Regarding what you said, that is intended to fix a bug with the send command. It was not working the way it was, as the signOperationEvent is expected not to be on raw version, but in normal version. I mentioned this to @haammar-ledger. Could you help me here Hakim?

Reply:
It replaces the signOperation object to its raw format here 
https://github.com/Zondax/ledger-live/blob/134355d561bd8d576123d51f99cb5058be5721a4/libs/ledger-live-common/src/transaction/signOperation.ts#L60-L65

Reply:
Hi @emmanuelm41, I remember that change yes, but I can't say I understood either why it was needed. :slightly_smiling_face: 

Testing locally, the cli `send` command currently works on my end.
As advised by @gre and since it is outside the scope of a coin integration, could you please consider opening a dedicated PR for this change, describing the issue it solves and how to reproduce it? Thanks.